tests/unit/nupic/research/spatial_pooler_unit_test.py

```diff
@@ -1191,11 +1189,6 @@ def testInitPermanence1(self):
     self.assertGreater(numcon, 0)
     self.assertLess(numcon, sp._numInputs)
 
-    minThresh = sp._synPermActiveInc / 2.0
-    connThresh = sp._synPermConnected
-    self.assertEqual(numpy.logical_and((perm >= minThresh),
-                                       (perm < connThresh)).any(), True)
```

Comment:
ah yeah. well unit testing the new rule in the c++ was a little easier because the connected synapses and unconnected synapses are initialized separately, so we can test them separately. since that's not the case here, and we aren't guaranteed that it will be _exactly_ half and half (only approximately), I'm not sure what the best way to test would be.